app/styles/_shame.scss

```diff
@@ -127,3 +127,31 @@
   background-color: #e8ebee;
   cursor: pointer;
 }
+
+/*
+ For readonly view of semantic forms
+*/
+.disabled input.au-c-input.is-disabled {
+  background-color: transparent;
+  color: black;
+  border: transparent;
+  padding-left: 0;
+}
+
+.disabled .ember-power-select-trigger[aria-disabled="true"] {
+  background-color: transparent;
+  border: transparent;
+  padding-left: 0 !important;
```

Comment:
This !important is sadly necessary, because it is also !important in ember-submission-form-fields